scripts/config

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source /usr/share/yunohost/helpers
+
+# Stop script if errors
+ynh_abort_if_errors
+
+# Import common fonctions
+source ./psql.sh
+source ./experimental_helper.sh
+source ./_common.sh
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+app=$YNH_APP_ID
```

Comment:
;)

Reply:
Thanks,

Also Fix

Reply:
Should be fixed.
Work on my side with yunohost 3.3